src/rest_vol_attr.c

```diff
@@ -117,6 +116,10 @@ RV_attr_create(void *obj, const H5VL_loc_params_t *loc_params, const char *attr_
 
     /* If this is a call to H5Acreate_by_name, locate the real parent object */
     if (H5VL_OBJECT_BY_NAME == loc_params->type) {
+        // TODO: Proper way to verify a plist?
+        if (H5I_INVALID_HID == loc_params->loc_data.loc_by_name.lapl_id)
```

Comment:
The proper way would probably be either to use H5Iobject_verify or H5Iis_valid. But since H5Iobject_verify can't currently be called on library types and H5Iis_valid may not always return the right result, this is probably fine for now.